www/js/diary/cards/TripCard.tsx

```diff
@@ -51,21 +48,42 @@ const TripCard = ({ trip }) => {
   const mapStyle = showAddNoteButton ? s.shortenedMap : s.fullHeightMap;
   return (
     <DiaryCard timelineEntry={trip} flavoredTheme={flavoredTheme} onPress={() => showDetail()}>
-      <View style={[cardStyles.cardContent, {flexDirection: 'row'}]}>
-        <IconButton icon='dots-horizontal' size={28}
+      <View style={[cardStyles.cardContent, {flexDirection: 'row-reverse'}]}
+        accessibilityLabel={`Trip from ${displayStartTime} to ${displayEndTime}`}>
+        <IconButton icon='dots-horizontal' size={24}
+          accessibilityLabel="View trip details" onPress={() => showDetail()}
           style={{position: 'absolute', right: 0, top: 0, height: 16, width: 32,
                   justifyContent: 'center', margin: 4}} />
-        <View style={{flex: 1}}>{/* left panel */}
+        <View style={s.rightPanel}>{/* right panel */}
+          <View style={[cardStyles.panelSection, {marginTop: 0}]}>{/* date and distance */}
+            <Text style={{fontSize: 14, textAlign: 'center'}}>
+              <Text style={{fontWeight: 'bold', textDecorationLine: 'underline'}}>{displayDate}</Text>
+            </Text>
+            <Text style={{fontSize: 13, textAlign: 'center'}}>
+              {t('diary.distance-in-time', {distance: formattedDistance, distsuffix: distanceSuffix, time: displayTime})}
+            </Text>
+          </View>
+          <View style={cardStyles.panelSection}>{/* start and end locations */}
+            <StartEndLocations displayStartName={tripStartDisplayName}
+              displayEndName={tripEndDisplayName} />
+          </View>
+          <View style={[cardStyles.panelSection, {marginBottom: 0}]}>{/* mode and purpose buttons / survey button */}
+            {surveyOpt?.elementTag == 'multilabel' &&
+                <MultilabelButtonGroup trip={trip} />}
+            {surveyOpt?.elementTag == 'enketo-trip-button'
+                && <UserInputButton timelineEntry={trip} />}
```

Comment:
It seems like it would be useful to have a component that is the equivalent of `linkedsurvey` that we could use in any location. Or is the thought that it was just an artifact of the diary/label dichotomy and we will always use the trip component with the labels even if we want to label trips elsewhere?

Reply:
My thinking was that it's only 4 lines of code and it's only used in 2 places (`TripCard` and `LabelDetailsScreen`)

Extracting this to a separate component would save 6 lines of code but add ~15-25 in a new file due to boilerplate